ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -123,10 +127,23 @@ public boolean valueContainsAny(final Set<ColumnName> names) {
    *
    * @param windowed indicates that the source is windowed; meaning {@code WINDOWSTART} and {@code
    * WINDOWEND} columns will added to the value schema to represent the window bounds.
+   *
+   * @param pseudoColumnVersion indicates the pseudocolumn version number of the given query.
+   * Queries passed to this method will receive the corresponding pseudocolumns in their schema.
+   *
    * @return the new schema.
    */
+  public LogicalSchema withPseudoAndKeyColsInValue(
+      final boolean windowed, final int pseudoColumnVersion) {
+    return rebuild(//this will use current pseudocolumn version when feature is fully released
```

Comment:
Rather than leaving comments like this, we should set the current pseudocolumn version to be the same as legacy version for now and update this to use the current pseudocolumn version instead of the legacy one.

Reply:
I miswrote this comment, this will actually use the passed `pseudoColumnVersion` parameter when the feature is released. How should I handle this given I can't just change the constant?